R/conditions.R

```diff
@@ -281,86 +354,90 @@ printStackTrace <- function(cond,
   full = get_devmode_option("shiny.fullstacktrace", FALSE),
   offset = getOption("shiny.stacktraceoffset", TRUE)) {
 
-  should_drop <- !full
-  should_strip <- !full
-  should_prune <- !full
-
-  stackTraceCalls <- c(
+  stackTraces <- c(
     attr(cond, "deep.stack.trace", exact = TRUE),
     list(attr(cond, "stack.trace", exact = TRUE))
   )
 
-  stackTraceParents <- lapply(stackTraceCalls, attr, which = "parents", exact = TRUE)
-  stackTraceCallNames <- lapply(stackTraceCalls, getCallNames)
-  stackTraceCalls <- lapply(stackTraceCalls, offsetSrcrefs, offset = offset)
+  dfs <- mapply(
```

Comment:
@cpsievert This could maybe be a simple for-loop instead? (Claude wrote the mapply)

Reply:
Oh yea, I think I like that marginally better

Reply:
I decided not to turn this into a for-loop after all; after fixing the stripStackTrace scoring issue (thanks for catching that, btw) the mapply is actually useful as we're now iterating over 3 things in parallel.

Reply:
Sounds good, thanks. In that case, maybe lets just change `dfs <- mapply(` to `mappy(` to make it more obvious that it's performing side-effects?